src/Components/SearchBar/index.jsx

```diff
@@ -20,10 +20,14 @@ const SearchBar = (props) => {
           placeholder="Search the word..."
           onFocus={showSuggestions}
           onBlur={hideSuggestions}
+          onChange={(e) => {
+            props.setSearchTerm(e.target.value);
+          }}
```

Comment:
Scope for simplification:
```suggestion
          onChange={(e) => props.setSearchTerm(e.target.value)}
```